modules/pensions/lib/pensions/monitor.rb

```diff
@@ -1,13 +1,15 @@
 # frozen_string_literal: true
 
-require 'zero_silent_failures/monitor'
+require 'logging/monitor'
 
 module Pensions
   ##
   # Monitor functions for Rails logging and StatsD
   # @todo abstract, split logging for controller and sidekiq
   #
-  class Monitor < ::ZeroSilentFailures::Monitor
+  class Monitor
+    include Logging::Monitor
+
```

Comment:
calling `super` when not inheriting is very odd.  shuffling around zsf and general monitor will need some changes here too

Reply:
This was an accident, I forgot to put back the inherit.